shell-completion/zsh/_systemd-analyze

```diff
@@ -87,6 +87,9 @@ _arguments \
     '--system[Operate on system systemd instance]' \
     '--user[Operate on user systemd instance]' \
     '--global[Show global user instance config]' \
+    '--error-on=[When verifying a unit, control dependency verification]:MODE' \
+    '--image=[Add support for discrete images]:PATH' \
+    '--root=[Add support for root arguments]:PATH' \
```

Comment:
--root and --image should go in the other PR (same for the other completion file)